recipes/icc/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "icc" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
```

Something like this should be fine
```suggestion
  # Prefix with "python-" to avoid the name colliding with the Intel C++ compiler
  name: python-{{ name|lower }}
```